lesson-1-1 submission

Tugeeta opened the pull request.

XeniyaDob reviewed (changes requested):
Hi Tugeeta,

Good job with creating react app! 

Please, make sure to test that the web page is working correctly by running npm start in the terminal.

I am getting an error, and if you can't figure out how to fix that, I encourage you to reach out to one of the mentors and have them go through the code with you.
It sucks sometimes, but debugging is a part of programming, no matter how good you are at it. You can do this!
<img width="751" alt="Screen Shot 2023-03-14 at 9 18 00 AM" src="https://user-images.githubusercontent.com/11164899/225016141-e5f30f83-bf3e-4dfd-9770-bee9ce4c853f.png">

Tugeeta commented:
Hi XeniyaDob,

Thank you for reviewing the project and letting me know my mistakes.  I
fixed it,  The array index is the problem.  I removed all the zeros from Id
and it worked.

Thanks and regards,


On Tue, Mar 14, 2023 at 9:30 AM XeniyaDob ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi Tugeeta,
>
> Good job with creating react app!
>
> Please, make sure to test that the web page is working correctly by
> running npm start in the terminal.
>
> I am getting an error, and if you can't figure out how to fix that, I
> encourage you to reach out to one of the mentors and have them go through
> the code with you.
> It sucks sometimes, but debugging is a part of programming, no matter how
> good you are at it. You can do this!
> [image: Screen Shot 2023-03-14 at 9 18 00 AM]
> <https://user-images.githubusercontent.com/11164899/225016141-e5f30f83-bf3e-4dfd-9770-bee9ce4c853f.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Tugeeta/react-todo/pull/1#pullrequestreview-1339286043>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ6ZZEJY22H3T3FPQQBM72TW4BXHFANCNFSM6AAAAAAVZ4J34E>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

Tugeeta committed:
lesson-1-1 submission
